modules/encryption.md

```diff
@@ -0,0 +1,71 @@
+# Encryption 
+
+The `encrypted` key is a Link Property.
```

Comment:
I feel like this page is missing a short leading paragraph explaining the purpose of this module.

Reply:
you're right.